sql/migrations/20210810191400_world.sql

```diff
@@ -0,0 +1,780 @@
+INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES
+(6767, 8080);
+
+-- Merithra of the Dream Gossip
+UPDATE `creature_template` SET `gossip_menu_id` = 6766, `npc_flags` = 3 WHERE `entry`=15378;
+INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES
+(6766, 8079);
+
+-- Guard Didier Gossip
+UPDATE `creature_template` SET `npc_flags` = 3 WHERE `entry` = 16226;
+INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES
+(7168, 8438);
+
+-- Elder Torntusk
+UPDATE `creature_template` SET `gossip_menu_id` = 6102 WHERE `entry` = 14757;
+INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES
+(6102, 7257);
+
+-- Kelm Hargunth
+UPDATE `creature_template` SET `gossip_menu_id` = 6141, `npc_flags` = 16391 WHERE `entry` = 14754;
+INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES
+(6141, 7294);
+INSERT INTO `gossip_menu_option` (`menu_id`, `id`, `option_icon`, `option_text`, `option_broadcast_text`, `option_id`, `npc_option_npcflag`) VALUES
+(6141, 0, 1, 'What goods have I earned the right to purchase for use in Warsong Gulch?', 10021, 3, 4);
+
+-- Primal Torntusk
+UPDATE `creature_template` SET `gossip_menu_id` = 6084 WHERE `entry` = 14736;
+INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES
+(6084, 7238);
+
+-- Corporal Noreg Stormpike
+UPDATE `creature_template` SET `gossip_menu_id` = 5081, `npc_flags` = 3 WHERE `entry`=13447;
+INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES
+(5081, 6288);
+
+-- Seersa Copperpinch, Guchie Jinglepocket and Jaycrue Copperpinch
+UPDATE `creature_template` SET `gossip_menu_id` = 5181 WHERE `entry` IN (13432, 13436, 13430);
+UPDATE `creature_template` SET `npc_flags` = 7 WHERE `entry` = 13432;
+UPDATE `creature_template` SET `npc_flags` = 5 WHERE `entry` = 13436;
+UPDATE `creature_template` SET `npc_flags` = 5 WHERE `entry` = 13430;
+
+-- Cyrus Therepentous
+UPDATE `creature_template` SET `gossip_menu_id` = 1841 WHERE `entry` = 9459;
+INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES
+(1841, 2493);
+
+-- Dreka'Sur
+UPDATE `creature_template` SET `gossip_menu_id` = 4067 WHERE `entry` = 9620;
+INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES
+(4067, 4960);
+
+-- Fix drop rate of items used in quest 5725 (The Power to Destroy...) in Ragefire Chasm (credit cmangos)
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = -5 WHERE `item` IN (14395, 14396);
+
+-- Fix drop rate of item used in quest 443 (Rot Hide Ichor) in Silverpine Forest (credit cmangos)
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = -30 WHERE `item` = 3236;
+
+-- Fix drop rate of items used in quest 451 (A Recipe For Death) in Silverpine Forest (credit cmangos)
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = -50 WHERE `item` = 3257;
+
+-- Correct Diplay ID For Energetic Rod
+UPDATE `item_template` SET `display_id` = 30915 WHERE `entry` = 18321;
+
+-- Remove Teebus Blazing Longsword From Lockboxes
+DELETE FROM `reference_loot_template` WHERE `entry` = 1014 AND `item`=1728;

```

Comment:
Should other weapons be removed too? Like Alcor's Sunrazor, Shadowblade and Blanchard's Stout. According to wowhead they aren't dropping either.